renderdoc/driver/d3d12/d3d12_pixelhistory.cpp

```diff
@@ -111,6 +112,30 @@ struct D3D12EventInfo
   uint8_t padding1[8];
 };
 
+struct D3D12PipelineReplacements
+{
+  ID3D12PipelineState *fixedShaderStencil;
+  ID3D12PipelineState *originalShaderStencil;
+};
+
+namespace
```

Comment:
Purely style query : any rationale or benefit for the anonymous namespace here? as opposed to a static function at file scope or a static helper function as part of the class

Reply:
Habit? :P There were a few symbol collisions with the Vukan impl (hence all the D3D12 prefixes sprinkled about), so some solution is beneficial. I'm happy to change it to the preferred approach, does the RenderDoc source have a style guideline for this?

Reply:
I would say leave it. Anonymous namespace for small number of items makes sense.